backend/llm/openai.py

```diff
@@ -80,11 +78,11 @@ def vector_store(self) -> CustomSupabaseVectorStore:
         )
 
     @property
-    def question_llm(self) -> LLM:
+    def question_llm(self):
```

Comment:
We may need to consider keeping the return type, as a forthcoming PR will enforce its mandatory usage

Reply:
This causes errors because ChatOpenAI is not the same type as LLM. Not sure how you think we should play that?